src/main/java/com/google/finapp/SpannerDaoInterface.java

```diff
@@ -0,0 +1,26 @@
+/*
+description goes here...
+ */
+
+package com.google.finapp;
+
+import com.google.cloud.ByteArray;
+import com.google.cloud.spanner.*;
+
+import java.math.BigDecimal;
+
```

Comment:
A couple of more changes which need to be made:

- In the server code where we inject SpannerDao, we need to instead inject SpannerDaoInterface.
- In the database module we should bind SpannerDaoInterface to the implementation - https://www.baeldung.com/guice#2-guice-bindings

Reply:
do you mean in `FinAppService` to change from the `SpannerDao` to `SpannerDaoInterface`?